docs/service_table.md

```diff
@@ -0,0 +1,16 @@
+# Service Roles
+
+| Service                                              | Language      | Description                                                                                                                       |
+| ---------------------------------------------------- | ------------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| [frontend](../src/frontend/README.md)                           | Go            | Exposes an HTTP server to serve the website. Does not require signup/login and generates session IDs for all users automatically. |
+| [cartservice](../src/cartservice/README.md)                     | C#            | Stores the items in the user's shopping cart in Redis and retrieves it.                                                           |
+| [productcatalogservice](../src/productcatalogservice/README.md) | Go            | Provides the list of products from a JSON file and ability to search products and get individual products.                        |
+| [currencyservice](../src/currencyservice/README.md)             | C++      | Converts one money amount to another currency. Uses real values fetched from European Central Bank. It's the highest QPS service. |
+| [paymentservice](../src/paymentservice/README.md)               | Node.js       | Charges the given credit card info (mock) with the given amount and returns a transaction ID.                                     |
+| [shippingservice](../src/shippingservice/README.md)             | Rust            | Gives shipping cost estimates based on the shopping cart. Ships items to the given address (mock)                                 |
+| [emailservice](../src/emailservice/README.md)                   | Ruby        | Sends users an order confirmation email (mock).                                                                                   |
+| [checkoutservice](../src/checkoutservice/README.md)             | Go            | Retrieves user cart, prepares order and orchestrates the payment, shipping and the email notification.                            |
+| [recommendationservice](../src/recommendationservice/README.md) | Python        | Recommends other products based on what's given in the cart.                                                                      |
+| [adservice](../src/adservice/README.md)                         | Java          | Provides text ads based on given context words.                                                                                   |
+| [featureflagservice](../src/featureflagservice/README.md)         | Erlang/Elixir | CRUD feature flag service to demonstrate various scenarios like fault injection & how to emit telemetry from a feature flag reliant service.                                             |
```

Comment:
Consider aligning the columns.

Reply:
Just updated it a bit using a vs code extension. Better but not 100% perfect